setup.md

```diff
@@ -4,166 +4,241 @@ This document will guide you through setting up Fennel on your
+    - [Using Fennel with LOVE](#using-fennel-with-love)
+- [Expanding your Fennel development experience](#expanding-your-fennel-development-experience)
+    - [Adding Fennel support to Emacs](#adding-fennel-support-to-emacs)
+        - [To add Fennel support to Emacs](#to-add-fennel-support-to-emacs)
+    - [Adding Fennel support to Vim](#adding-fennel-support-to-vim)
+        - [To add Fennel support to Vim](#to-add-fennel-support-to-vim)
+    - [Adding Fennel support to Neovim](#adding-fennel-support-to-neovim)
+        - [To add Fennel support to Neovim](#to-add-fennel-support-to-neovim)
+    - [Adding readline support to Fennel](#adding-readline-support-to-fennel)
+        - [To add readline support to Fennel](#to-add-readline-support-to-fennel)
+
+# Requirements
 
 * Access to a UNIX-like environment, such as Ubuntu, Debian, Arch
   Linux, Windows Subsystem for Linux, Homebrew, scoop.sh, etc.
 * Lua version 5.1, 5.2, 5.3, or LuaJIT
 * LuaRocks or Git
 
-Note that Fennel can be used on non-Unix systems, but that is out of
-scope for this document.
+# Downloading and installing Fennel
 
-# Downloading Fennel
-
-Currently, you can either download Fennel from
-[https://github.com/bakpakin/Fennel](https://github.com/bakpakin/Fennel)
-or use LuaRocks to download Fennel. On certain operating systems, you
-might have Fennel available from your system package manager, but that is
-out of scope for this document.
+Downloading and installing Fennel on your system allows you to run
+Fennel scripts. Currently, you can download and install Fennel using
+Git or LuaRocks.
 
 Depending on which method you want to use, choose a subsection below:
 
 * [Using Git to download Fennel](#using-git-to-download-fennel)
 * [Using LuaRocks to download Fennel](#using-luarocks-to-download-fennel)
 
-**Tip**: If you are using an application that already has Fennel
-built-in (such as [TIC-80](https://tic.computer)) you might not need
-to download Fennel at all.
+**Tip**: If you are using software that supports Fennel, such as
+[TIC-80](https://tic.computer), you do not need to download Fennel,
+because you can use it inside of TIC-80.
+
+## Using Git to download and install Fennel
+
+Downloading and installing Fennel using Git allows you to use versions
+of Fennel that haven't been released yet and makes contributions to
+Fennel easier.
 
-## Using Git to download Fennel
+### To download Fennel
 
-You may want to use Git to install if:
+1. `cd` to a directory in which you want to download Fennel, such as
+   `~/src`
+2. Run `git clone https://github.com/bakpakin/Fennel`
 
-* You want to use a version that hasn't been released yet
-* You want to contribute changes to Fennel
-* You have Git installed and don't want to bother with another system
+### To install Fennel
 
-To install:
+1. Run `cd Fennel`
+2. Run `make fennel`
+3. Copy or link `fennel` to a location on your shell's `$PATH`, such
+   as `~/bin` or `/usr/local/bin`
```

Comment:
This seems to imply that `~/bin` is on the shell's `PATH` in most situations, which I don't believe is the case unless the user has already customized their `PATH` to include it. If the user has a customized `PATH`, they probably know where their custom bin directory is. I think we can remove `~/bin` here. `/usr/local/bin` makes sense to keep IMO tho.

Reply:
Good call! This was me carrying out examples from the pre-edited version, and I wasn't sure if it was safe to remove.